### PR TITLE
nmcli: refresh

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
I don't feel like I have done a good job here, sorry that this might take a little longer to get sorted

File: pages/linux/nmcli.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
 # nmcli
 
 > A command-line tool for controlling NetworkManager.
-> More information: <https://man.archlinux.org/man/nmcli.1>.
+> This is a forwarding page to the advanced commands of `nmcli`.
+> More information: <https://manned.org/nmcli>.
 
-- Check the nmcli version:
+- Run an `nmcli` command:
```
Comment (navarroaxel) [inline pages/linux/nmcli.md:7]: Is this too generic?

**Review by navarroaxel — COMMENTED** [review]
I think we should add:
```markdown
- Display help for a subcommand:
- 
nmcli {{subcommand}} --help
```

File: pages/linux/nmcli.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
 # nmcli
 
 > A command-line tool for controlling NetworkManager.
-> More information: <https://man.archlinux.org/man/nmcli.1>.
+> This is a forwarding page to the advanced commands of `nmcli`.
```
Comment (sbrl) [inline pages/linux/nmcli.md:4]: I'm not sure this line is needed.

File: pages/linux/nmcli.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
 
-`nmcli --version`
-
-- Call general help:
-
-`nmcli --help`
+`nmcli {{agent|connection|device|general|help|monitor|networking|radio}} {{command_options}}`
 
-- Call help on a command:
+- Display `nmcli` version:
```
Comment (sbrl) [inline pages/linux/nmcli.md:11]: This isn't grammatically correct.
```suggestion
- Display the current version of NetworkManager:
```

Comment (CleanMachine1) [reply pages/linux/nmcli.md:11]:
```suggestion
- Display the current version of `nmcli:
```

Comment (navarroaxel) [reply pages/linux/nmcli.md:11]:
```suggestion
- Display version of NetworkManager:
```

File: pages/linux/nmcli.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/linux/nmcli.md:7]: nmcli doesnt do anything without the subcommand, I think its fine

Comment (marchersimon) [reply pages/linux/nmcli.md:7]: How about
```suggestion
- Run an `nmcli` subcommand:
```

File: pages/linux/nmcli.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/nmcli.md:4]: I don't think so either

**Review by marchersimon — APPROVED** [review]
Looks good

File: pages/linux/nmcli.md [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
+- Display the current version of NetworkManager:
 
-`nmcli {{command}} --help`
+`nmcli --version`
 
-- Execute an `nmcli` command:
+- Display help:
 
-`nmcli {{command}}`
+`nmcli --help`
```
Comment (navarroaxel) [inline pages/linux/nmcli.md:16]:
```suggestion
`nmcli --help`

- Display help for a subcommand:

`nmcli {{subcommand}} --help`
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Pull request merged**